tornado_mysql/converters.py

```diff
@@ -123,7 +123,7 @@ def convert_datetime(obj):
         usecs = '0'
         if '.' in hms:
             hms, usecs = hms.split('.')
-        usecs = float('0.' + usecs) * 1e6
+        usecs = round(float('0.' + usecs) * 1e6)
```

Comment:
@dangnvang 

i think that any of the changes should be fine, i added a new scenario and it looks ok as well :slightly_smiling_face:

This test code:
```
new2_errors = 0

for expected in range(1000000):
    usecs_string2 = round(float('0.' + str(expected).zfill(6)) * 1e6)

    if usecs_string2 != expected:
        new2_errors += 1

print 'new2_errors: ' + str(new2_errors)
```

Throws this output
```
new2_errors: 0
```

So i think both cases are ok, what do you think?